shoop/front/settings.py

```diff
@@ -63,3 +63,16 @@
 #:
 #: Handled error cases are: 400, 403, 404, and 500
 SHOOP_FRONT_INSTALL_ERROR_HANDLERS = True
+
+#: A dictionary defining properties to override the default field properties in the
+#: checkout address form. For example, this can be used to set ``required`` to
+#: ``True`` to force a certain field to be required.
+#:
+#: The dictionary should map the field name (as a string) to a dictionary of the
+#: overridding properties.
+#:
+#: It should be noted, however, that overriding some settings (such as making a
+#: required field non-required) could create other validation issues.
+SHOOP_FRONT_ADDRESS_FIELD_PROPERTIES = {
+    "postal_code": {"required": True}
```

Comment:
Instead of postal code I was thinking to add fields that is required before this change. Which would be name, street, city and country.